parity/rpc_apis.rs

```diff
@@ -647,6 +658,7 @@ impl ApiSet {
 				public_list
 			},
 			ApiSet::SafeContext => {
+				public_list.insert(Api::Debug);
```

Comment:
Why enable this only for safe context (given that bad blocks cache is enabled by default)? I understand that the `debug` namespace might have other methods in the future which are unsafe to expose.

Reply:
Didn't enable for two reasons:
1. `debug` is potentially unsafe
2. It should never be used by regular dapps
3. We have this implemented to support detecting forks, I don't want to make it a first-class namespace where we implement all the methods, cause that most likely won't happen - the methods are not going through EIPs, and they are pretty much geth-specific debug RPCs.